File: python/sort/selection.sort.py
arr = [40,00,302,502,424,1048,948]

size = len(arr)


print("Original array is: ",arr)

for i in range(size):
    index = i
    
    for j in range(index,size):
        if arr[index] > arr[j]:
            index = j
    arr[index],arr[i] = arr[i],arr[index]
    
    
print("Sorted array is: ",arr)

File: js/Sort/selection.sort.js
const arr = [20,50,103,40,3050,20,450,230]


console.log("original  array is: ",arr)

for(let i = 0; i<arr.length; i++){
    let index = i;
    for(let j = index; j<arr.length; j++){
        if(arr[index] > arr[j]){
            index = j;
        }
    }

    [arr[i],arr[index]] = [arr[index],arr[i]]
}

console.log("Sorted array is: ",arr)